infra/terraform/test-org/github/labels.tf

```diff
@@ -26,96 +26,21 @@ locals {
       color : "d73a4a"
       description : "Something isn't working"
     },
-    {
-      name : "duplicate",
-      color : "cfd3d7"
-      description : "This issue or pull request already exists"
-    },
-    {
-      name : "good first issue",
-      color : "7057ff"
-      description : "Good for newcomers"
-    },
-    {
-      name : "help wanted",
-      color : "008672",
-      description : "Extra attention is needed"
-    },
-    {
-      name : "invalid",
-      color : "e4e669",
-      description : "Something doesn't seem right"
-    },
     {
       name : "question",
```

Comment:
I don't think this provides too much value

Reply:
SG - removed